R/mainTests.r

```diff
@@ -967,29 +979,18 @@ overcompleted_cols <- function(data, meta) {
 
   overcomplete_regional_cols <- function(matrixRow) {
     # Start by filtering the data down to remove the geographic level being tested and any lower levels we don't care about
+    levels_incompatible_with_region <- c("National", "Local skills improvement plan area")
 
     level_rows <- data %>%
-      filter(geographic_level != matrixRow[1]) %>%
-      filter(!geographic_level %in% geography_matrix[3:16, ])
+      filter(geographic_level %in% levels_incompatible_with_region)
```

Comment:
Should we still be filtering out the matrixRow level too? It was before but isn't now?

Reply:
This is filtering it out, it's only _keeping_ levels that are in the incompatible_with_region vector.